pkg/shell/util.tsx

```diff
@@ -19,11 +19,11 @@
 
 import cockpit from "cockpit";
 
-/* Encode navigate state into a string If with_root is true the
- * configured url root will be added to the generated url. with_root
- * should be used when navigating to a new url or updating history,
- * but is not needed when simply generating a string for a link.
- */
```

Comment:
This comment removal lives in "shell: Move util.jsx to TypeScript" but really belongs into the previous commit which changed the function signature already.

Reply:
Done.